src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsProvider.java

```diff
@@ -444,4 +442,21 @@ private Set<NodeCandidate> getPaidNodeCandidate(Infrastructure infra, String reg
         }).collect(Collectors.toSet());
         return result;
     }
+
+    /**
+     * @return The Operating system family based on the infrastructure type, created to patch the limitation caused
+     * by Jclouds for OpenStack (not taking the distro_family into account).
```

Comment:
The comment could be a bit more clear. 
e.g., Retrieve the operating system family. When the infrastructure type is not openstack-nova, directly retrieved it from image metadata. Otherwise, we need to get it from XXXX because of the jclouds limitaion.

Reply:
done